scipy/signal/_filter_design.py

```diff
@@ -290,15 +290,15 @@ def freqz(b, a=1, worN=512, whole=False, plot=None, fs=2*pi,
     Parameters
     ----------
     b : array_like
-        Numerator of a linear filter. If `b` has dimension greater than 1,
-        it is assumed that the coefficients are stored in the first dimension,
-        and ``b.shape[1:]``, ``a.shape[1:]``, and the shape of the frequencies
-        array must be compatible for broadcasting.
+        Numerator of a linear filter. Multidimentional inputs are not supported,
+        and the shape of the frequencies array must be compatible for broadcasting.
+        If `b` has dimension greater than 1, use `scipy.signal.freqz(np.zeros((128, 10, 1)))` or
+        b[..., np.newaxis] to adjust the dimension.
```

Comment:
I am not really sure what is meant by the `scipy.signal.freqz(np.zeros((128, 10, 1)))` comment, can you clarify?